### DISCUSSION TITLE
Changing the screen timeout

### DISCUSSION THREAD

**Discussion by iu14472** [tanujnotes/Olauncher#484] [Ideas]
The ability to set a screen timeout after returning from any application to 3, 2, or 1 second (can be configured in settings).

**Comment by tutacat** [comment]
Use Automate to do this. First you want to save the before timeout to a file, then when olauncher id opens you retrieve the latest song, set to x settings, then when you leave olauncher, you set back to the previous setting. 

I say to a file or something because you want it to be consistent if you restart while olauncher on screen it would have x seconds as the system timeout instead of the current setting. 

This isn't something a normal launcher does.

Or on Samsung you can use Routines to set timeout to (15) seconds while olauncher is open and revert when closed.

On Samsung at least, you must turn off "keep screen on while viewing" for that to work as intended.
